pkgs/top-level/python-packages.nix

```diff
@@ -7519,6 +7519,8 @@ self: super: with self; {
     protobuf = protobuf3;
   };
 
+  onnx-graphsurgeon = callPackage ../development/python-modules/onnx-graphsurgeon { };
+
```

Comment:
Could you reword this commit and pull title into `python3Packages.onnx-graphsurgeon: init at 0.3.25` ?

Reply:
Updated. Thanks for the help!